### ISSUE TITLE
What about the `healthcare` tag?

### ISSUE THREAD

**Issue by PaulUithol** [hotosm/health#1]
http://wiki.openstreetmap.org/wiki/Key:healthcare

Seems to be quite well specced

**Comment by PaulUithol** [comment]
Also includes `healthcare:specialty`. American Red Cross has been using this tagging in West Africa

**Comment by Matthias84** [comment]
Indeed, the healthcare 2.0 is still an draft, while the healthcare=* is accepted by vote.
